### ISSUE TITLE
Keyboard shortcut collision

### ISSUE THREAD

**Issue by Cyborg572** [rmariuzzo/cloud9-emmet-ext#4]
The keyboard shortcut: <kbd>CTRL</kbd> + <kbd>SHIFT</kbd> + <kbd>E</kbd> is the same shortcut cloud 9 uses for it's "Quick Outline" feature, (the one that lets you jump to a function/variable/class declaration by typing it's name)

**Comment by rmariuzzo** [comment]
Good catch! Any suggestion by the comment I can map the expand command to <kbd>CTRL</kbd> + <kbd>ENTER</kbd>.

**Comment by Cyborg572** [comment]
As far as I can tell that one's free

**Comment by rmariuzzo** [comment]
Great I will change that default key binding. Thanks for notifying this.

**Comment by jordanboston** [comment]
This change does not seem to be implemented yet.  Another thing that works great in Sublime Text with Emmet is that when you expand abbreviation the cursor lands between the tags instead of at the end.  This really makes a difference when coding IMO.

So if I had an h1 and then expanded it I would have 
  `<h1>|</h1>`  rather than `<h1></h1>|`

Thanks

**Comment by rafaelrinaldi** [comment]
Are we able to run actions throught `tab` key? Instead of a shortcut. That would be rad.
